web3.js/test/connection.test.ts

```diff
@@ -545,6 +545,51 @@ describe('Connection', () => {
     }
   });
 
+  it('get inflation reward', async () => {
+    await mockRpcResponse({
+      method: 'getInflationReward',
+      params: [
+        [
+          '7GHnTRB8Rz14qZQhDXf8ox1Kfu7mPcPLpKaBJJirmYj2',
+          'CrinLuHjVGDDcQfrEoCmM4k31Ni9sMoTCEEvNSUSh7Jg',
+        ],
+        {
+          epoch: 0,
+        },
+      ],
+      value: [
+        {
+          amount: 3646143,
+          effectiveSlot: 432000,
+          epoch: 0,
+          postBalance: 30504783,
+        },
+        null,
+      ],
+    });
+
+    let inflationReward;
+    try {
+      inflationReward = await connection.getInflationReward(
+        [
+          new PublicKey('7GHnTRB8Rz14qZQhDXf8ox1Kfu7mPcPLpKaBJJirmYj2'),
+          new PublicKey('CrinLuHjVGDDcQfrEoCmM4k31Ni9sMoTCEEvNSUSh7Jg'),
+        ],
+        0,
+      );
+    } catch (error) {
+      // a live server needs to have slots up to 432000
+      // in order to have rewards for epoch 0
+      if (!mockServer && error.message.match(/Block not available/)) {
```

Comment:
how about only running this test as a mock then?

Reply:
+1
However, solana-test-validator now supports `--slots-per-epoch`, so if there are similar tests in the future that are important to run live, we could boot the CI test validator with shorter epochs.

Reply:
Nice, good to know @CriesofCarrots.